<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>para_wlop</class>
 <widget class="QWidget" name="para_wlop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WLOP parameters setting</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>338</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Radius:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radius">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mu">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sample Average Mu3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sample_average_mu3">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Iterate </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="iter">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>times</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="Need_sample_average">
      <property name="text">
       <string>Need Sample Average</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Need_original_combine_sample">
      <property name="text">
       <string>Need Original Combine Sample</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Need_averaging_movement">
      <property name="text">
       <string>Need Averaging Movement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="compute_density">
      <property name="text">
       <string>Compute Density</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="compute_pca">
      <property name="text">
       <string>Compute PCA Normal Each Iteration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Use_Elliptical_Original_Neighbor">
      <property name="text">
       <string>Use_Elliptical_Original_Neighbor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Use_Adaptive_Mu">
      <property name="text">
       <string>Use_Adaptive_Mu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Use_KNN_Sample_Neighbor">
      <property name="text">
       <string>Use_Adaptive_Sample_Neighbor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Use_tangent_vector">
      <property name="text">
       <string>Use_tangent_vector</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Use_confidence">
      <property name="text">
       <string>Use_confidence</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="anisotropic_lop_apply">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Anisotropic LOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wlop_apply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>WLOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dual_wlop_apply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dual WLOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="step_forward">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wlop_projection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Projection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dual_connection">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dual Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skel_wlop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Skel WLOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dual_drag_wlop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dual Drag WLOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regularize_samples">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Regularize Samples</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regularize_normals">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Regularize Normals</string>
   </property>
  </widget>
  <widget class="QPushButton" name="normal_reform">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal reform</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compute_confidence">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute Confidence</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
